ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -19,8 +19,8 @@ set -o nounset
 set -o pipefail
 
 # Export proxy to ensure commands like curl could work
-export HTTP_PROXY=${HTTP_PROXY}
-export HTTPS_PROXY=${HTTPS_PROXY}
+[ ! "x${HTTP_PROXY}" == "x" ] && export HTTP_PROXY=${HTTP_PROXY}
+[ ! "x${HTTPS_PROXY}" == "x" ] && export HTTPS_PROXY=${HTTPS_PROXY}
```

Comment:
Kube generally prefers the style:
```
[[ -n "${HTTP_PROXY:-}" ]]
```

Doesn't your version hit the -o nounset ?